bentoml/cli/deployment.py

```diff
@@ -250,28 +258,40 @@ def get(name, namespace, output):
         default=ALL_NAMESPACE_TAG,
     )
     @click.option(
-        '--limit', type=click.INT, help='Limit how many resources will be retrieved'
+        '-p',
+        '--platform',
+        type=click.Choice(['aws-sagemaker', 'aws-lambda']),
```

Comment:
should we use 'sagemaker' and 'lambda' to match the commands? we will need to do a mapping from cli to the operator type name in proto tho